.mergify.yml

```diff
@@ -8,14 +8,11 @@ pull_request_rules:
       merge:
         strict: smart
         method: squash
-  # This rule keep the PR updated with its base branch as soon as it has 1
-  # approval. This makes it more likely to be ready to be merged once the
-  # second approval comes.
   - name: prepare for merge
     conditions:
       - label!=manual merge
       - check-success=test
-      - "#approved-reviews-by>=1"
+      - "#approved-reviews-by>=2"
     actions:
       update: {}
```

Comment:
You can remove this rule altogether then, the `strict: smart` does it automatically.